tests/DeepCopyTest/DeepCopyTest.php

```diff
@@ -385,6 +387,18 @@ public function test_it_uses_the_first_filter_matching_for_copying_object_proper
         $this->assertNull($copy->getAProp()->cloned);
     }
 
+    public function test_it_can_deep_copy_an_array_object()
```

Comment:
Just to be sure: this test fails on PHP 7.4 if the new `ArrayObjectFilter` is not registered?

Reply:
Yes, without `ArrayObjectFilter` `$foo` and `$copy['foo']` would be same.